_posts/dart-cuts.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: "Background on DART cuts"
 excerpt: "At the time of writing this, 5 cities have voted for and 2 cities have voted against a reduction in DART's funding by 25%. Here's how we got here and what's coming next."
-coverImage: "/assets/blog/dart-cuts/cover.jpg"
+coverImage: "/assets/blog/dart-cuts/dartCuts.png"
```

Comment:
everything is dash-case so I think this and the other file names should also file that

Reply:
Sorry I grabbed the direct file from Connor and didn't rename it

Reply:
Still need to rename this to `dart-cuts.png`, same with some of the other images

Reply:
Files were renamed in the latest commit